pulseaudio-control.bash

```diff
@@ -38,6 +45,31 @@ function getCurVol() {
 }
 
 
+# Saves the `device.description` property of the sink passed by parameter into a variable named `curDescr`.
+function getCurDescr() {
+  curDescr=$(pacmd list-sinks | sed -n -e '/index: '"$1"'/,/index/ p' | grep 'device.description' | sed 's/.*"\(.*\)"/\1/g')
+}
+
+
+# Saves the name of the sink passed by parameter into a variable named `curName`.
+function getCurName() {
+  curName=$(pacmd list-sinks | sed -n -e '/index: '"$1"'/,/index/ p' | grep 'name:' | sed 's/.*<\(.*\)>/\1/g')
+}
+
+
+# Saves a `nickname` for the sink passed by parameter into a variable called `curNick`.
+# If a mapping for the sink name exists, that is used. Otherwise, the sinks description is used as fallback.
+function getCurNick() {
+  getCurName $1
```

Comment:
Could you please check the linter's messages and fix them? They're quite simple, you should just add double quotes for the parameters:

https://travis-ci.com/github/marioortizmanero/polybar-pulseaudio-control/builds/153408539#L237

Reply:
done